### ISSUE TITLE
Can't open Mac file.

### ISSUE THREAD

**Issue by dosch** [keeporsweep/keeporsweep.net#4]
Archive app gives an error.

![screen shot 2018-08-15 at 09 07 08](https://user-images.githubusercontent.com/5372610/44135728-a1d57172-a06a-11e8-93eb-dff861f936e9.png)

**Comment by jancborchardt** [comment]
Yes, this is sort of known, as I did the macOS packaging some time ago on a different laptop and didn’t really know which file to upload. I also didn’t work on the desktop version of the app for some time now so it will be some time before I fix it.

We are working on a rewritten web version though, and maybe I’ll just use Electron to use that same code to work as a desktop app. :) Will let you know.

**Comment by jancborchardt** [comment]
Also, while you are waiting, the Nextcloud app works fine! :) https://apps.nextcloud.com/apps/keeporsweep

If you like it, a rating is very welcome! :heart:

**Comment by dosch** [comment]
Yeah, I found the nextcloud app; works good indeed!
The thing is; my mac is full of clutter and this seems a nice way to go through it every now and then.
What I store in Nextcloud is very tidy, because of storage constrains... so the app is less usefull there.

One suggestion: display some meta-data next to a file; path/to/file, creation data, last opened etc. This helps a lot to determine if I should keep or sweep it. :-)
